tests/test_empiricals.py

```diff
@@ -79,14 +80,11 @@ def test_getPlasmaPauseErrors2(self):
         self.assertRaises(KeyError, spam1)
 
     def test_getPlasmaPauseCA1992warn(self):
-        with warnings.catch_warnings(record=True) as w:
-            warnings.simplefilter('always', category=RuntimeWarning)
+        with spacepy_testing.assertWarns(
+                self, 'always',
+                r'No LT dependence currently supported for CA1992 model',
```

Comment:
Just to confirm, these need to be raw strings because they get passed to a regex in the class? I mean, it all seems to work, but I haven't made sure yet that they're all actually raw strings. Likewise, I don't think I saw docs saying the strings passed here need to be passed as raw strings.

Reply:
[There's no such thing as a raw string](https://stackoverflow.com/a/2081708); "raw string" is sloppy colloquialism for "[raw string literal](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals)":
```python
raw = r'\foo'
print(type(raw)) #<type 'str'>
print(repr(raw)) #'\\foo'
cooked = '\\foo'
print(type(cooked)) #<type 'str'>
print(repr(cooked)) #'\\foo'
print(raw == cooked) #True
```

I tend to use raw literals whenver I do a regular expression just so I only have to worry about escaping the regex and not escaping the string as well. Even if there's nothing that technically needs escaping in there, I might come back and add something later and get bit. I also find it's a bit of a hint to the reader that this thing is a regex.

But the caller can construct their strings however they like.